content/release_notes.md

```diff
@@ -468,15 +512,15 @@ By default, Chef Infra Client eagerly loads all ruby files in each cookbook's li
 
 ```ruby
 eager_load_libraries false # disable eager loading all libraries
-eager_load_libraries 'helper_library.rb' # eager load just the the file helper_library.rb
+eager_load_libraries 'helper_library.rb' # eager load just the file helper_library.rb
 eager_load_libraries %w(helper_library_1.rb helper_library_2.rb) # eager load both helper_library_1.rb and helper_library_2.rb files
 ```
 
 Note: Unless you are experiencing performance issues in your libraries, we advise against changing the loading behavior.
 
 #### always_dump_stacktrace client.rb option
 
-A new `always_dump_stacktrace` client.rb configuration option and command line option allows you to have any Ruby stacktraces from Chef Infra Client logged directly to the log file. This may help troubleshooting when used in conjunction with centralized logging systems such as Splunk. To enable this new option, run `chef-client --always-dump-stacktrace` or add the following to your `client.rb`:
+A new `always_dump_stacktrace` client.rb configuration option and command-line option allows you to have any Ruby stacktraces from Chef Infra Client logged directly to the log file. This may help troubleshooting when used in conjunction with centralized logging systems such as Splunk. To enable this new option, run `chef-client --always-dump-stacktrace` or add the following to your `client.rb`:
```

Comment:
```suggestion
A new `always_dump_stacktrace` client.rb configuration option and command line option allows you to have any Ruby stacktraces from Chef Infra Client logged directly to the log file. This may help troubleshooting when used in conjunction with centralized logging systems such as Splunk. To enable this new option, run `chef-client --always-dump-stacktrace` or add the following to your `client.rb`:
```